recaf-ui/src/main/java/me/coley/recaf/util/JFXInjection.java

```diff
@@ -129,35 +124,6 @@ private static void addToClasspath(List<Path> dependencyPaths) {
 		}
 	}
 
-	/**
-	 * Create a visible alert that the user cannot install JavaFX automatically due to incompatible Java versions.
-	 */
-	private static void alertPre11UserMissingJFX() {
-		Toolkit toolkit = Toolkit.getDefaultToolkit();
-		toolkit.beep();
-		// Collect debug information
-		StringWriter writer = new StringWriter();
-		RuntimeProperties.dump(writer);
-		String debugInfo = writer.toString();
-		// Show message
-		String style = "<style>" +
-				"p {font-family: Arial; font-size:14;} " +
-				"pre { background: #DDDDDD; padding: 5px; border: 1px solid black; }" +
-				"</style>";
-		String message = "<p>The required JavaFX classes could not be found locally.<br><br>Your environment:</p>" +
-				"<pre>" + debugInfo + "</pre>" +
-				"<p>You have two options:<br>" +
-				" 1. Use a JDK that bundles JavaFX<br>" +
-				" 2. Update to Java 11 or higher <i>(Recaf will automatically download JavaFX)</i></p>";
-		JEditorPane pane = new JEditorPane("text/html", style + message);
-		pane.setEditable(false);
-		pane.setOpaque(false);
-		JOptionPane.showMessageDialog(null,
-				pane, "JavaFX could not be found",
-				JOptionPane.ERROR_MESSAGE);
-		System.exit(0);
-	}
-
 	/**
 	 * Create a visible alert that the user cannot install JavaFX automatically due to some error that occurred.
 	 */
```

Comment:
Eventually we will be able to remove this whole class :)

Reply:
Wasn't it required for users that run the jar without a launcher?

Reply:
Ah crap, I almost forgot...